src/LanguageServer/Impl/Implementation/EditorFiles.cs

```diff
@@ -143,7 +143,7 @@ internal sealed class EditorFile {
                 }
             } finally {
                 if (enqueueForParsing) {
-                    _server.UpdateSymbolIndexFromDoc(uri, doc);
+                    _server.RunningDocumentIndexer.UpdateDocument(uri, doc);
```

Comment:
I don't think name related to the RDT since indexer works on all files, not just running ones :-). So just `WorkspaceIndexer` will do.